pkg/api/releases.go

```diff
@@ -185,50 +184,53 @@ func GetPayloadAnalysis(dbc *db.DB, release, stream, arch string, numPayloadsToA
 	// sort so the most likely blocker test failures are first in the slice:
 	sort.Slice(testFailures, func(i, j int) bool { return testFailures[i].BlockerScore >= testFailures[j].BlockerScore })
 	result.TestFailures = testFailures
-	return result, nil
+	return result.TestFailures, nil
 }
 
 // calculateBlockerScore uses the list of most recent failed payloads, and compares to the failures we found
 // for a particular test, then attempts to calculate a blocker score between 0.0 (not a blocker) and 1.0 (almost
 // certainly a blocker) based on a number of criteria.
+//
+// consecutiveFailedPayloadTags is the list of our current streak of rejected payload tags. If most recent payload
+// was accepted, this list will be empty, and we don't have much processing to do.
 func calculateBlockerScore(consecutiveFailedPayloadTags []string, ta *apitype.TestFailureAnalysis) {
 	if len(consecutiveFailedPayloadTags) == 0 {
 		// our most recent state is Accepted, could be intermittent, but for the purposes of a blocker
 		// we have to assume 0.
-		ta.BlockerScore = 0.0
+		ta.BlockerScore = 0
```

Comment:
Suggest adding a blocker score reason explaining the above comment.  Currently we render a tooltip but without a reason.